subprojects/core-api/build.gradle.kts

```diff
@@ -59,7 +59,8 @@ strictCompile {
     ignoreRawTypes() // raw types used in public API
 }
 
-integTest.usesJavadocCodeSnippets = true
+// AutoTestedSamplesCoreApiIntegrationTest including customized test logic
```

Comment:
```suggestion
// AutoTestedSamplesCoreApiIntegrationTest includes customized test logic, so automatic auto testing samples generation is not needed (and would fail) in this project
```